manual/src/refman/extensions/effects.etex

```diff
@@ -43,40 +36,25 @@ We can handle the "Xchg" effect by implementing a handler that always returns
 the successor of the offered value:
 
 \begin{caml_example}{verbatim}
-try_with comp1 ()
-{ effc = fun (type a) (eff: a t) ->
-    match eff with
-    | Xchg n -> Some (fun (k: (a, _) continuation) ->
-        continue k (n+1))
-    | _ -> None }
+try comp1 () with
+| effect (Xchg n), k -> continue k (n+1)
 \end{caml_example}
 
-"try_with" runs the computation "comp1 ()" under an effect handler that handles
-the "Xchg" effect. As mentioned earlier, effect handlers are a generalization
-of exception handlers. Similar to exception handlers, when the computation
-performs the "Xchg" effect, the control jumps to the corresponding handler.
-However, unlike exception handlers, the handler is also provided with the
-delimited continuation "k", which represents the suspended computation between
-the point of "perform" and this handler.
+We run the computation "comp1 ()" under an effect handler that handles the
+"Xchg _" effect with a continuation bound to "k". Here "effect" is a keyword
+which signifies that this pattern matches effects and not exceptions. As
+mentioned earlier, effect handlers are a generalization of exception handlers.
+Similar to exception handlers, when the computation performs the "Xchg" effect,
```

Comment:
You changed `Xchg` to `Xchg _` a few lines above, would it make sense to change here too?

Reply:
If we are to be consistent, we should also change `Send` and `Fork`. What do you think @Octachron? In the papers, I've used `Xchg` effect not `Xchg _` effect. Would you say the latter is more correct?

Reply:
I think the `the effect "Xchg"` is a better term: `Xchg _` is a pattern. Keeping the two separated is probably clearer. But we could inline the next `this pattern` in the following sentence:

> We run the computation "comp1 ()" under an effect handler that handles the
"Xchg" effect with a continuation bound to "k". Here "effect" is a keyword
which signifies that the `Xchg n` pattern matches effects and not exceptions.

Reply:
Done.